test/docker-compose.yml

```diff
@@ -198,16 +135,75 @@ services:
     restart: always
     depends_on:
       - kafka
+    env_file:
+      - ./.env.test
     environment:
       PORT: 3000
-      KAFKA_BOOTSTRAP_SERVERS: kafka:9093
       EXTRACTOR_URL: http://extractor:3001
       TRANSFORMER_URL: http://transformer:3002
     ports:
       - "3001:3000"
 
+  pyrog-server:
+    image: arkhn/pyrog-server:latest
+    container_name: pyrog-server
+    expose:
+      - 1000
+    env_file:
+      - ./.env.test
+    environment:
+      JWT_PRIVATE_KEY: ${AUTH_SIGNING_PRIVATE_KEY}
+      JWT_TOKEN: ${FHIR_API_TOKEN}
+      PORT: 1000
+    volumes:
+      - ./volumes/mapping.json:/app/mapping.json
+      - ./volumes/credentials.json:/app/credentials.json
+    depends_on:
+      - postgres
+      - redis
+
+  postgres:
+    image: postgres
+    container_name: postgres
+    command: -p 55432
+    ports:
+      - 55432:55432
+    env_file:
+      - ./.env.test
+    volumes:
+      - test-postgres-data:/var/lib/postgresql/data
+
+  redis:
+    image: redis:5.0.7-alpine
+    container_name: redis
+    expose:
+      - 6379
+    volumes:
+      - redis-data:/data
+
+  fhir-api:
+    image: arkhn/fhir-api:latest
+    container_name: fhir-api
+    expose:
+      - 2000
+    ports:
+      - 2000:2000
+    env_file:
+      - ./.env.test
+    environment:
+      ES_HOST: http://elasticsearch:9200
+      JWT_PUBLIC_KEY: ${AUTH_SIGNING_PUBLIC_KEY}
+    depends_on:
+      - mongo
+      - elasticsearch
+    volumes:
+      - ./volumes/conceptMaps.json:/app/conceptMaps.json
```

Comment:
I would mount this volume to /fhir_bundles/conceptMaps.json instead of /app

Reply:
It's actually `/srv/flask_app/fhir_bundles` so I hesitated to use it because if it changes in the api's image, we need not to forget that we use it here